app/src/ui/add-repository/licenses.ts

```diff
@@ -74,6 +74,14 @@ function replaceTokens(
   return newBody
 }
 
+function ensureLineEndingSet(body: string) {
+  if (body.endsWith('\n')) {
```

Comment:
Using ternary `?` operator instead of the if conditional check makes it clean. Just thoughts :thinking:

Reply:
:+1: